src/topology/algebra/infinite_sum.lean

```diff
@@ -689,6 +703,9 @@ variables {f g : β → α} {a a₁ a₂ : α}
 lemma has_sum_le (h : ∀b, f b ≤ g b) (hf : has_sum f a₁) (hg : has_sum g a₂) : a₁ ≤ a₂ :=
 le_of_tendsto_of_tendsto' hf hg $ assume s, sum_le_sum $ assume b _, h b
 
+@[mono] lemma has_sum_mono (hf : has_sum f a₁) (hg : has_sum g a₂) (h : f ≤ g) : a₁ ≤ a₂ :=
```

Comment:
This for a later PR, but it looks like some more of the lemmas in this section could be renamed to let us take advantage of dot notation.